.github/workflows/sycl-containers.yaml

```diff
@@ -45,6 +45,12 @@ jobs:
             file: ubuntu2204_intel_drivers
             tags: ubuntu2204_intel_drivers
             build_args: "use_latest=false"
+          - name: Intel Drivers Ubuntu 22.04 Docker image with dev IGC
+            file: ubuntu2204_intel_drivers
+            tags: ubuntu2204_intel_drivers_devigc
+            build_args: |
+                  "use_latest=false"
+                  "use_igc_dev=true"
```

Comment:
@jsji, please, document new image in https://github.com/intel/llvm/blob/sycl/sycl/doc/developer/DockerBKMs.md#sycl-containers-overview.

Reply:
Done in https://github.com/intel/llvm/pull/13307